File: C++/FizzBuzz.cpp
//Implementing FizzBuzz in C++ to print Fizz Buzz in range of 1 to 50.
//Author: Keshav Kumar

#include <iostream>
using namespace std;
int main()
{
    for (int i = 0; i <=50; i++)
    {
        if (i%3==0 && i%5==0)
        {
            cout<<"FizzBuzz"<<endl;
        }
        else if (i%5==0)
        {
            cout<<"Buzz"<<endl;
        }
        else if(i%3==0){
            cout<<"Fizz"<<endl;
        }
        else{
            cout<<i<<endl;
        }
    }
    return 0;
}

File: Python/FizzBuzz.py
#iterating over 1 to 50 using range 
for i in range(1,51):
    #checking if i is divisible by 3
    if i%3==0:
        #printing Fizz as i is divisible by 3
        print("Fizz")
    #checking if i is divisible by 5
    elif i%5==0:
        #printing Fizz as i is divisible by 3
        print("Buzz")
        
    else:
        #printing i as i is not divible by either 3 or 5 
        print(i)
